src/utils/spawning.js

```diff
@@ -0,0 +1,68 @@
+const body = require("utils_body_constants");
+const roles = require("utils_role_constants");
+var createName = require("utils_naming")
+
+var spawner = Game.spawns['Gubbins'];
```

Comment:
We should raise an issue to move spawner name to config, and then we should do some thinking to get multi-spawner stuff working. I'll do that now.

Reply:
/issues/11

Reply:
/issues/12